plugins/fetch.json

```diff
@@ -1,66 +1,66 @@
 {
-    "cordova-plugin-network-information": {
-        "source": {
-            "type": "registry",
-            "id": "cordova-plugin-network-information@~1.3.2"
-        },
-        "is_top_level": true,
-        "variables": {}
+  "cordova-plugin-network-information": {
```

Comment:
For any other reviewers, jsondiff.com tells me that "The two files (are) semantically identical."